ironfish/src/wallet/walletdb/accountValue.ts

```diff
@@ -37,13 +38,18 @@ export class AccountValueEncoding implements IDatabaseEncoding<AccountValue> {
     flags |= Number(!!value.spendingKey) << 0
     flags |= Number(!!value.createdAt) << 1
     flags |= Number(!!value.multiSigKeys) << 2
+    flags |= Number(!!value.proofAuthorizingKey) << 3
     bw.writeU8(flags)
     bw.writeU16(value.version)
     bw.writeVarString(value.id, 'utf8')
     bw.writeVarString(value.name, 'utf8')
     if (value.spendingKey) {
       bw.writeBytes(Buffer.from(value.spendingKey, 'hex'))
     }
+
+    if (value.proofAuthorizingKey) {
+      bw.writeBytes(Buffer.from(value.proofAuthorizingKey, 'hex'))
+    }
```

Comment:
I think this is fine, but I was initially uneasy about adding this in the middle of the serialization.

Since the migration will be reading the old version and overwriting with the new version I don't think we'll see any problems